examples/bert/openvino/bert_base_multilingual_cased/requirements.txt

```diff
@@ -0,0 +1,4 @@
+datasets
+nncf==2.16.0
+optimum-intel
```

Comment:
What's the difference between this and `optimum[openvino]`? We can simplify this as well like my comment in https://github.com/microsoft/Olive/pull/1786